app/lib/template-updater/template-updater.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+/**
+ * @typedef {import('./types').Endpoint} Endpoint
+ * @typedef {import('./types').TemplateUpdateResult} TemplateUpdateResult
+ */
+
+const { EventEmitter } = require('node:events');
+
+const Queue = require('../util/queue');
+
+const { updateTemplates } = require('./util');
+
+const log = require('../log')('app:template-updater');
+
+/** @type {Endpoint} */
+const OOTB_CONNECTORS_ENDPOINT = {
+  executionPlatform: 'Camunda Cloud',
+  fileName: '.camunda-connector-templates.json',
+  url: 'https://marketplace.cloud.camunda.io/api/v1/ootb-connectors'
+};
+
+module.exports.OOTB_CONNECTORS_ENDPOINT = OOTB_CONNECTORS_ENDPOINT;
+
+/** @type {Endpoint[]} */
+const DEFAULT_ENDPOINTS = [
```

Comment:
Should we remove this? We seem to explicitly pass it anyway: https://github.com/camunda/camunda-modeler/pull/5119/files#diff-db39e3ab79485fbcb1c55605a8d40585fec91ecf66fc61b602f7967076610de1R758

Reply:
Done https://github.com/camunda/camunda-modeler/pull/5119/commits/10919cc507d020c4a7f68cf7289f7eb526ef9617